### DISCUSSION TITLE
InputText input size & weight

### DISCUSSION THREAD

**Discussion by mauricekindermann** [filamentphp/filament#10295] [Ideas]
This seems like a useful feature for inputs and would be consistent with the $table options

![Screenshot 2023-12-14 at 11 48 47 am](https://github.com/filamentphp/filament/assets/3828998/687c740f-5f07-4a3e-ad05-d68537db0420)

```php
->size('lg')
->weight('bold')
```

I have checked the docs and it doesn't seem to exist, so for now I hack it in:

```php
->extraInputAttributes(['style' => 'font-size:1.25em;padding:0.75em;font-weight:bold;'])
```

**Comment by MtDalPizzol** [comment]
Good feature to have. Sometimes we need a more compact styling....
